src/main/java/net/commments/Test22.java

```diff
@@ -0,0 +1,42 @@
+package net.commments;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class Test22 {
+
+    private final WebDriver[] driver = new WebDriver[1];
+
+    @Test
+    public void test() {
+        this.driver().get("http://commentssprintone.azurewebsites.net");
+        this.driver().findElements(By.id("newbutton")).get(0).click();
+        WebElement text = this.driver().findElement(By.id("Text"));
```

Comment:
Please join all lines 21-23 into one. It's a bad practice to declare a variable if it's used only once.

Reply:
@extsoft done